docs/examples/visualizer/index.html

```diff
@@ -0,0 +1,550 @@
+            width: 30%;
+        }
+    </style>
+</head>
+
+<body>
+    <p class='start'>
+        <h4>CM-Well graph explorer</h4>
+        <label for="starting">Starting point:</label><input id="starting" type="text" size="30" value='/permid.org/1-4295905573'><button onclick="reset(document.getElementById('starting').value)">Get Data</button><br/>
+    </p>
+    <div id="details_popup">
+        <table>
+            <tr>
+                <th>Input</th>
+                <th>Action</th>
+            </tr>
+            <tr>
+                <td>What is this?</td>
+                <td>Demonstrates how to connect D3 and CM-Well</td>
+            </tr>
+            <tr>
+                <td>Input box</td>
+                <td>Put any subject URI and app will try and retrieve from host</td>
+            </tr>
+            <tr>
+                <td>Mouse Wheel</td>
+                <td>Zoom</td>
+            </tr>
+            <tr>
+                <td>Drag (white space)</td>
+                <td>Pan</td>
+            </tr>
+            <tr>
+                <td>Hover</td>
+                <td>Show node details</td>
+            </tr>
+            <tr>
+                <td>Click</td>
+                <td>Attempt to load the node from host</td>
+            </tr>
+
+        </table>
+    </div>
+    <div id='resize_container'>
+        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <defs>
+            <marker id='arrowhead' viewbox='0 -5 10 10' refX='15' refY='0' orient='auto' markerWidth='10' markerHeight='10'>
+                <path d='M 0,-5 L 10 ,0 L 0,5' fill='#bbb' stroke= '#bbb'></path>
+            </marker>
+        </defs>
+      </svg>
+    </div>
+    <script>
+        /**
+            Convert a URI to a full URL request (using an anchor tag to do the parsing)
+            The function supports hosting out of localhost (e.g without CM-Well and just static JSON)
+        **/
+        function formatRequestURL(requested_uri) {
+            parser = document.getElementById('url_parser');
+            parser.href = requested_uri;
+            var return_uri = window.location.protocol + "//";
+            if (parser.hostname === "localhost") { // we have the whole fragment in the pathname
+                return_uri += window.location.host + parser.pathname;
+            } else if (parser.hostname === window.location.hostname) {
+                return_uri += window.location.host + parser.pathname;
+            } else {
+                return_uri += window.location.host + "/" + parser.hostname + parser.pathname;
+            }
+            if (window.location.hostname !== "localhost") { // e.g. we're hosted on CM-Well
+                return_uri += "?format=jsonl";
+            }
+            return return_uri;
+        }
+
+        /**
+            Don't render system infotons
+        **/
+        function isSystemInfoton(uri) {
+            if (String(uri).endsWith(".sys"))
```

Comment:
Sorry for being petty, but please refactor to `return String(uri).endsWith(".sys"))` (no need for an `if` here)

Reply:
Duh. I'm embarrassed by this. Thanks for keeping me honest.